js/src/components/paid-ads/campaign-assets-form.js

```diff
@@ -77,6 +80,14 @@ export default function CampaignAssetsForm( {
 
 	const [ baseAssetGroup, setBaseAssetGroup ] = useState( initialAssetGroup );
 	const [ hasImportedAssets, setHasImportedAssets ] = useState( false );
+	const [ countryCodes, setCountryCodes ] = useState( [] );
+	const { validateCampaignWithCountryCodes } =
+		useValidateCampaignWithCountryCodes( countryCodes );
+
+	const handleOnChange = ( _, values, arg ) => {
+		setCountryCodes( values.countryCodes );
```

Comment:
@eason9487 The main reason why I had a state variable for `countryCodes` is because we need to know when the selected countries are updated so that we can fetch the latest recommendations. I've changed the approach a bit by having the state variable within the `useValidateCampaignWithCountryCodes` where you can pass updated country codes via the `refreshCountryCodes` function from the hook.